site/en/tutorials/load_data/images.ipynb

```diff
@@ -380,7 +380,8 @@
         "id": "2ZgIZeXaDUsF"
       },
       "source": [
-        "The `image_batch` is a tensor of the shape `(32, 180, 180, 3)`. This is a batch of 32 images of shape `180x180x3` (the last dimension referes to color channels RGB). The `label_batch` is a tensor of the shape `(32,)`, these are corresponding labels to the 32 images. \n"
+        "The `image_batch` is a tensor of the shape `(32, 180, 180, 3)`. This is a batch of 32 images of shape `180x180x3` (the last dimension referes to color channels RGB). The `label_batch` is a tensor of the shape `(32,)`, these are the corresponding labels to the
+        32 images. \n"
```

Comment:
I think this is breaking the JSON. Can you remove the extra newline? Thanks

Reply:
Let me fix it